web/client/components/styleeditor/config/wellKnownName.js

```diff
@@ -9,7 +9,7 @@
 const wellKnownName = [
     {
         value: 'Circle',
-        label: 'styleeditor.circle',
+        label: 'Circle',
```

Comment:
did we stopped localizing these strings?

Reply:
It was never localized so we could use a simple label because of the wellKnownName property